.eslintrc.js

```diff
@@ -15,6 +15,7 @@ module.exports = {
   rules: {
     /* || Eslint main rules || */
     camelcase: [2, { properties: 'always' }], // Override airbnb style.
+    'consistent-return': 'off',
```

Comment:
I'm not sure about this one. So I'm utilizing the cleanup pattern for React's `useEffect` hook [here](https://github.com/department-of-veterans-affairs/vets-website/pull/21315/files#diff-e903d0c8bd5b5b9f67bc6110669e234e55b186ea59ed68cf8545d947d4be8c42).

By using the returned cleanup function, the linter was failing here, but it shouldn't be. I definitely don't want to disable the rule, and then for every other instance like this. And generally, I'd prefer to keep code consistent by applying the appropriate rules on the linter config by enforcing them across all files, and almost never disable a rule for the same reason. Send me a DM if you want to converse more quickly about this (when you're back online.)

Reply:
I see why the linter was failing - the conditional was returning before the cleanup function could return, which is the only thing that should be returned, otherwise `void` should be returned which is no return at all. Removing this disabled rule.